src/Malawi_cor_labels_GEE script.js

```diff
@@ -0,0 +1,197 @@
+var malawi_binary = ee.Image("projects/ee-aasareansah/assets/malawi_binary"),
+    Sentinel2 = ee.Image("projects/ee-aasareansah/assets/Sent_Sept_Comp_Malawi"),
+    Malawi = ee.FeatureCollection("FAO/GAUL/2015/level0").filter(ee.Filter.eq('ADM0_NAME', 'Malawi')),
+    Malawi_EPA = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_EPAs"),
```

Comment:
To keep it general, you don't need to name these variables Malawi
Also I have no idea what `EPA` and `TA` means could these acronyms be converted to fuller variable names?

Reply:
Yes, I will remove these layers from the general code. EPA and TA stand for Extension Planning Area and Traditional Authority respectively.